firmware/baseband/proc_btlerx.cpp

```diff
@@ -275,12 +507,14 @@ void BTLERxProcessor::on_message(const Message* const message) {
 }
 
 void BTLERxProcessor::configure(const BTLERxConfigureMessage& message) {
-    (void)message;  // avoid warning
+    channel_number = message.channel_number;  // avoid warning
```

Comment:
warning?

Reply:
Oh, I see - delete that comment.
Also, if you see that pattern in the future in a method you touch, just delete the parameter name in the function signature if it's unused. No sense in naming something just so we can add a hack-around to keep the compiler from complaining about the name. :)